editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -1061,132 +1061,134 @@ void CanvasItemEditor::_switch_theme_preview(int p_mode) {
 	EditorNode::get_singleton()->update_preview_themes(theme_preview);
 }
 
+// Check if we are hovering a guide.
```

Comment:
This comment is unnecessary, because it repeats method's name.
Also it's misleading. If we were checking for hover, the method would return a bool, but from its usage it clearly does something. I think it should be renamed.